sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CustomCoder.java

```diff
@@ -22,19 +22,14 @@
 import java.util.List;
 
 /**
- * An abstract base class for writing a {@link Coder} class that encodes itself via Java
- * serialization.
+ * An abstract base class for writing a {@link Coder} class.
```

Comment:
This sentence, the topic paragraph, is now tautological. It should either say something about the nature of the abstract base class or be deleted.